library/proc_macro/src/bridge/buffer.rs

```diff
@@ -39,7 +39,7 @@ pub struct Buffer<T: Copy> {
     data: *mut T,
     len: usize,
     capacity: usize,
-    extend_from_slice: extern "C" fn(Buffer<T>, Slice<'_, T>) -> Buffer<T>,
+    reserve: extern "C" fn(Buffer<T>, usize) -> Buffer<T>,
```

Comment:
This is the last use of `Slice` and now `Slice` can be removed (not sure why it didn't warn/error as unused).